modules/userId/userId.md

```diff
@@ -111,6 +111,12 @@ pbjs.setConfig({
                  name: '_criteoId',
                  expires: 1
               }
+        }, {
+            name: "cpexId",
+            storage: {
+                type: "cookie",
+                name: "caid"
+            }
```

Comment:
this isn't necessary based on your implementation, since you manually pull from browser storage.